packages/mobile/src/navigator/types.tsx

```diff
@@ -89,9 +93,11 @@ export type StackParamList = {
     recipient: Recipient
   }
   [Screens.JoinCelo]: undefined
-  [Screens.Language]: {
-    nextScreen?: keyof StackParamList
-  }
+  [Screens.Language]:
+    | {
```

Comment:
prettier is weird eh?

Reply:
indeed

Reply:
It is actually pretty cool! I think they took this syntax from Elm 👍